Tensile/Source/lib/include/Tensile/PlaceholderLibrary.hpp

```diff
@@ -38,6 +38,26 @@ namespace Tensile
     // To be extended in the future
     enum class LazyLoadingInit
     {
+        None,
+        gfx803,
+        gfx900,
+        gfx906,
+        gfx908,
+        gfx90a,
+        gfx940,
+        gfx941,
+        gfx942,
+        gfx1010,
+        gfx1011,
+        gfx1012,
+        gfx1030,
+        gfx1031,
+        gfx1032,
+        gfx1034,
+        gfx1035,
+        gfx1100,
+        gfx1101,
+        gfx1102,
         All
     };
```

Comment:
Was there a way to unify this with the other architecture enum that already exists?